### ISSUE TITLE
SdlRemoteDisplay subclass missing second ACTION_UP event

### ISSUE THREAD

**Issue by Tnnnguyen** [smartdevicelink/sdl_android#787]
### Bug Report
During multi-touch operations, when there are 2 finger touchdowns on the head unit, the MyDisplay (subclassing SdlRemoteDisplay) class's child view correctly receives 2 ACTION_DOWN events.  But when both fingers lift off, only 1 ACTION_UP event is received. This applies for 3 or more simultaneous touch events: all ACTION_DOWN events appear to be sent but only one ACTION_UP.

##### Reproduction Steps
1. Connect project app to head unit
2. Set the target child view's onTouchListener, overriding onTouch(view, eventMotion) method
3. Place 2 fingers on head unit's screen on the child view, observe 2 ACTION_DOWN events received in MyDisplay.java child view's onTouch(view, motionEvent) method.
4. Lift both fingers (at the same time or one after another)

##### Expected Behavior
Two ACTION_UP events delivered to MyDisplay's child view onTouch(view, motionEvent) method.

##### Observed Behavior
Only one ACTION_UP events delivered

##### OS & Version Information
* Android Version: Android 7
* SDL Android Version: 4.5

**Comment by joeygrover** [comment]
ACTION_UP only occurs once the last pointer is released so only one should be sent. [Per Android documentation](https://developer.android.com/training/gestures/multi)

Are you listening for ACTION_POINTER_UP?

**Comment by Tnnnguyen** [comment]
The fact that we only create motion events with ACTION_DOWN, ACTION_UP, ACTION_MOVE in `SdlProxyBase.convertTouchEvent` and the moment the first finger is up ACTION_UP is already sent and received throw me off so I was not listening for ACTION_POINTER_DOWN or ACTION_POINTER_UP.

I also notice during 2 fingers touch, the first finger up triggers the event from RPC notification -> `SdlProxyBase.convertTouchEvent` -> `remoteDisplay.handleMotionEvent` -> `mainView.dispatchTouchEvent` -> MyDisplay receiving. When the second finger is up, the same sequence happens but MyDisplay does not receive anything.  

Additionally, upon receiving ACTION_DOWN for more than one touch, the pointer id is the same for all touchdowns in onTouch() of my OnTouchListener.

**Comment by joeygrover** [comment]
Ok, we'll take a look

**Comment by brettywhite** [comment]
Hey @Tnnnguyen , 

I've been playing with this and I cannot reproduce what you are referring to. I have tried from 1 all the way up to 5 fingers and always receive the corresponding `ACTION_UP` to every `ACTION_DOWN`. I am using a TDK with S3 V3. I used the android sample streaming app and tested with SDL Android 4.5 and the current develop branch on a Pixel2XL running O. 

Are you using a TDK? If so maybe try another one? If not, try one. I cannot see any issues with the reading / converting of touch events in the library.

thanks
